### ISSUE TITLE
Amethyst is not working on Mojave 1.14.1

### ISSUE THREAD

**Issue by egel** [ianyh/Amethyst#765]
## System

 - **Amethyst version**: 0.12.1 (61)
 - **macOS version**: 1.14.1

## What's the problem?
Amethyst is not working (not resizing the windows).

## How can it be reproduced?
- Install MacOSX 1.14.1 
- get amethyst 0.12.1 (61)

> Sidenote: My current setup involve 3 additional screens. But before this bug appears the Amethyst works perfectly fine, even with multiple screens.

## What applications are involved?
All of them. It does not resize anymore any application in my system.

## Has anything fixed it, even temporarily?
Unfortunately, there was nothing. Although, before the update to v1.14.1 of MacOSX, the version 1.14.0 works fine, so I can only say this might be because of a new update to MacOSX Mojave.

## dotfile (if you have one)
Don't have any.

## Screenshots
![out](https://user-images.githubusercontent.com/3322106/48828908-42a50b00-ed71-11e8-9fb3-5e40a4975e78.gif)

![screenshot 2018-11-21 at 10 04 31](https://user-images.githubusercontent.com/3322106/48830650-6407f600-ed75-11e8-928b-104e96229ada.png)

**Comment by egel** [comment]
Yes @ianyh,  the new develop version (0.12.2b4 (65)) worked, but only for a couple of days (1-4) for me. Since 2-3 days after I installed develop version Amethyst is not working and:
- disabling/enabling not solving the problem
- relaunch amethyst (the button in the context menu) not solving the problem
- killing the app in activity monitor and restarting not solving the problem
- restarting the system also not helps

**Comment by kylechadha** [comment]
I was able to fix this by removing Amethyst from Security & Privacy accessibility settings, and then re-adding it.

**Comment by egel** [comment]
Hey @kylechadha, thank you for post. Unfortunatelly, your solution did not work for me as well.

I’m wondering if there are any hidden Amethyst setting which are added somewhere during first instalation. 

PS. Using mojave 1.14.3.

**Comment by egel** [comment]
After so many days without Amethyst, I finally found the solution to my problem of being not able to make it working again 🤦‍♂️🥳. The steps were the following: 

1. I've uninstalled Amethyst (from _brew_ and as well all previously download precompiled version)

1. I've searched all occurrences of `*Amethyst*` pattern across all filenames or directories in my system and removed all suspected files

        sudo find / -name "*Amethyst"

1. I've reinstalled Amethyst via `brew cask install amethyst`

I hope this will help somebody.

> My current version:
> - macOS: 10.15.1
> - Amethyst: 0.14.3 (79)

**Comment by mkniewallner** [comment]
Just for information, cask provides a `zap` command that formulas can implement so that all the files associated to a package also get removed when uninstalling a package (see https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md). You can do `brew cask zap amethyst`. The list of files that gets deleted for amethyst are listed in the brew formula (https://formulae.brew.sh/api/cask/amethyst.json).
